src/Cake.NuGet.Tests/Fixtures/NuGetContentResolverFixture.cs

```diff
@@ -28,6 +28,7 @@ protected NuGetContentResolverFixture(string framework, Runtime runtime)
             Environment = FakeEnvironment.CreateUnixEnvironment();
             Environment.Runtime.BuiltFramework = new FrameworkName(framework);
             Environment.Runtime.Runtime = runtime;
+            Environment.Runtime.IsCoreClr = runtime == Runtime.CoreClr;
 
```

Comment:
Why is it that the `FakeEnvironment` has a setter for this? Wouldn't it be beneficial to just check `Runtime == Runtime.CoreClr`?

I can fix it in a separate PR.